joomla/joomla-cms
# [5.1] Generic doc icon

Replaces the pdf icon used in media manager with a more generic doc icon as this is applied to a variety of doc types.

### To test
Use one of the prebuilt packages for this PR and check to see that the icon used in media manager for any doc type is no longer a pdf icon.

Changing the icon to a specific one for each type of doc is beyond the scope of this PR

Pull Request for Issue #42518

## Before
![image](https://github.com/joomla/joomla-cms/assets/1296369/e7002373-5b88-4336-8b7d-587830c136bf)

## After
![image](https://github.com/joomla/joomla-cms/assets/1296369/e1e5ca67-5393-4aa8-81c9-f62b0788dfa4)
 .

### Link to documentations
Please select:
